README.md

```diff
@@ -19,7 +21,7 @@ Here we have an overview of how you can start integrating this SDK into your app
          android:value="@string/admob_app_id" />
          
 ## 4.  Add Admob test-Id in strings class: 
-    <string name="admob_app_id">ca-app-pub-3940256099942544~3347511713</string>
+    <string name="admob_app_id">ca-app-pub-XXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
```

Comment:
@aqibbangash We've hard coded this app Id here, as this exact id is recommended for use by Google itself, for testing AdMob integration. The integration may not work as intended if use a blank/empty Id.